### DISCUSSION TITLE
How could I get the size of each request and response?

### DISCUSSION THREAD

**Discussion by chrismaresca** [jdepoix/youtube-transcript-api#361] [Q&A]
Looking for a simple way to get the size of each request/response. I noticed that implementing the service on a major cloud provider is causing blacklisted IPs so going to try and implement rotating proxies but want to be able to estimate the number of requests I could make per GB. Any help is appreciated!

**Comment by FVTAL** [comment]
I used WARP on digital ocean server which avoids the issue you're experiencing. Will likely receive recaptcha requests which show as 400 if the IP is being rejected. That or check logs for rate limiting and implement way to handle. Otherwise Cloudflare's WARP could likely help your issue.
